src/iov.c

```diff
@@ -37,8 +37,8 @@
 #include <ofi.h>
 #include <ofi_iov.h>
 
-uint64_t ofi_copy_iov_buf(const struct iovec *iov, size_t iov_count, uint64_t iov_offset,
-			  void *buf, uint64_t bufsize, int dir)
+uint64_t ofi_copy_iov_buf(const struct iovec *iov, size_t iov_count, size_t iov_offset,
```

Comment:
Probably a good idea to change the return type as well. I wasn't able to find any context wrt why `uint64_t` was chosen over `size_t`, but it's likely an oversight considering the compatibility.

Reply:
Makes sense to me. Let me give it a try.